src/lib/api.js

```diff
@@ -82,8 +116,8 @@ const api = {
      * @return {Promise} - HTTP response
      */
     get(url, { type: responseType = 'json', ...options } = {}) {
-        return api.xhr(url, { method: 'get', responseType, ...options });
-    },
+        return this.xhr(url, { method: 'get', responseType, ...options });
```

Comment:
I don't think we use `call` or `apply` anywhere for API methods, but probably a good idea to confirm the context here is always what we expect.